extensions/Worker.Extensions.Http/HttpTriggerAttribute.cs

```diff
@@ -67,5 +91,37 @@ public HttpTriggerAttribute(AuthorizationLevel authLevel, params string[] method
         /// Gets the authorization level for the function.
         /// </summary>
         public AuthorizationLevel AuthLevel { get; private set; }
+
+        private static AuthorizationLevel GetAuthorizationLevel(string authLevel)
+        {
+            if (string.IsNullOrWhiteSpace(authLevel))
+            {
+                return AuthorizationLevel.Function;
+            }
+
+            var trimmed = authLevel.Trim();
+            if (string.IsNullOrWhiteSpace(trimmed))
+            {
+                return AuthorizationLevel.Function;
+            }
+
+            if (!trimmed.StartsWith("%"))
+            {
+                return AuthorizationLevel.Function;
+            }
+
+            if (!trimmed.EndsWith("%"))
+            {
+                return AuthorizationLevel.Function;
+            }
```

Comment:
These two conditional are a single prerequisite - if it's a parameterized variable, `%` should appear at the beginning and at the end. Collapsing the two checks into a single if statement would keep them together.

```suggestion
            if ((trimmed.StartsWith("%") && trimmed.EndsWith("%")) is false)
            {
                return AuthorizationLevel.Function;
            }
```

Reply:
Thanks for the comment. It's updated.